src/tomography/tensornetwork-statetomography.jl

```diff
@@ -403,7 +403,7 @@ end
 Compute the gradients of the cost function:
 `C = log(Z) - ⟨log P(σ)⟩_data`
 """
-function gradients(L::LPDO, data::Matrix{Pair{String,Int}}; sqrt_localnorms=nothing)
+function gradients(L::LPDO, data::Matrix{Pair{String,Int}}; sqrt_localnorms=nothing, kwargs...)
```

Comment:
Why is this `kwargs` needed?

Reply:
This is because the `gradients` function for process tomography has an extra argument `trace_preserving_regularizer` wrt state tomography, and I have unified the `tomography` function, which now works for both state and process.